gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -13,7 +13,7 @@ try:
 {# NOTE: `pragma: NO COVER` is needed since the coverage for presubmits isn't combined. #}
 except ImportError as e:  # pragma: NO COVER
     {# TODO(https://github.com/googleapis/google-auth-library-python/pull/1577): Update the version of google-auth once the linked PR is merged. #}
-    raise ImportError("async rest transport requires google.auth >= 2.35.0 with aiohttp extra. Install google-auth with the aiohttp extra using `pip install google-auth[aiohttp]==2.35.0`.") from e
+    raise ImportError("async rest transport requires google-auth >= 2.35.0 with aiohttp extra. Install google-auth with the aiohttp extra using `pip install google-auth[aiohttp]==2.35.0`.") from e
```

Comment:
Please add a TODO bug to add an `aiohttp` extra for GAPICs and then update this error message to say install the extra

Reply:
Filed and linked https://github.com/googleapis/gapic-generator-python/issues/2187.